docs-src/docs/guides/walkthrough/deployment-1.md

```diff
@@ -0,0 +1,489 @@
+# Deployment 1: A Simple Guestbook App
+
+In this section you'll learn how different Layer0 commands work together to deploy applications to the cloud.
+The example application in this section is a guestbook -- a web application that acts as a simple message board.
+You can choose to complete this section using either [the Layer0 CLI](#deploy-with-layer0-cli) or [Terraform](#deploy-with-terraform).
+
+
+---
+
+## Deploy with Layer0 CLI
+
+If you're following along, you'll want to be working in the `walkthrough/deployment-1/` directory of your clone of the [guides](https://github.com/quintilesims/guides) repo.
```

Comment:
I'm not sure there's a whole lot more explicitness to get from this; `walkthrough` is a first-level directory of `guides`:

```
guides/
  + guestbook/
  + one-off-task/
  + terraform-beyond-layer0/
  + walkthrough/
  + windows/
```

I could probably say "work in `guides/walkthrough/deployment-1`", but I feel like "... of your clone of the guides repo" covers that.

Of course, I can't give a full system path because OS's are different, and we don't really care (and can't predict) _where_ on their system they clone the guides repo. Do you have a specific thought I should pursue?